ocaml/idl/datamodel_pool.ml

```diff
@@ -1136,6 +1136,40 @@ let set_ext_auth_max_threads =
     ~params:[(Ref _pool, "self", "The pool"); (Int, "value", "The new maximum")]
     ~allowed_roles:_R_POOL_OP ()
 
+let set_ext_auth_cache_enabled =
+  call ~flags:[`Session] ~name:"set_ext_auth_cache_enabled" ~lifecycle:[]
```

Comment:
Based on recent discussion: is the `~flags` argument not the default such that this parameter could be dropped?